src/main/python/rlbot/agents/base_agent.py

```diff
@@ -96,6 +98,10 @@ def get_field_info(self):
         This does not change during a match so it only needs to be called once after the everything is loaded."""
         return self.__field_info_func()
 
+    def get_physics_tick(self) -> PhysicsTick:
```

Comment:
Considering we just added a native version for ball prediction we might need that for physics as well.

If the point of this is instead for ML recording (which this would be great for) it might be good to do the "saving" of the data in the interface dll.

Reply:
I assume by "native" you mean C structs – Tare's on it.

Saving the data can happen wherever anyone likes, that can be a separate PR.